libs/blocks/preflight/accessibility/check-video-captions.js

```diff
@@ -0,0 +1,38 @@
+import { getConfig, getFederatedContentRoot } from '../../../utils/utils.js';
+
+export default async function checkVideoCaptions(elements = [], config = {}) {
+  if (!config?.checks?.includes('video-captions')) return [];
+  const violations = [];
+  const mcpIframes = elements.filter((el) => el.tagName.toLowerCase() === 'iframe' && el.src.includes('video.tv.adobe.com'));
+  const federalRoot = getFederatedContentRoot();
+  const { locale } = getConfig();
+  const geo = (locale?.prefix || '').replace('/langstore', '').replace('/', '');
+  const localesMap = await fetch(`${federalRoot}/federal/preflight/preflight-config.json?sheet=captions-map`)
+    .then((r) => r.json())
+    .then((data) => data.data.map((l) => [l.captions, l.geos.split(',')]));
+  const captionsLang = localesMap.find(([, geos]) => geos.includes(geo))?.[0];
+  for (const iframe of mcpIframes) {
+    const videoId = iframe.src.match(/\/v\/(\d+)/)?.[1];
+    // eslint-disable-next-line
```

Comment:
Why does eslint complain here.. can we fix the issue rather than disabling the rule?

Reply:
`continue` has always been disabled, I'd also suggest switching the logic to using `return` or splitting in more methods.